DirectProgramming/DPC++/N-BodyMethods/Nbody/src/GSimulation.cpp

```diff
@@ -0,0 +1,230 @@
+//==============================================================
+// Copyright © 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+// =============================================================
+// Copyright (c) 2019 Fabio Baruffa
+// Source: https://github.com/fbaru-dev/particle-sim
+// MIT License
+// =============================================================
+
+#include "GSimulation.hpp"
+// dpc_common.hpp can be found in the dev-utilities include folder.
```

Comment:
The default style is there should be a blank line between the include and the comment

Reply:
This change is done and the commit is pushed to this PR